test/services/sync/strategies/LDAPSyncerConsumer.integration.test.js

```diff
@@ -0,0 +1,280 @@
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+
+const { LDAP_SYNC_ACTIONS } = require('../../../../src/services/sync/strategies/LDAPSyncer');
+const { LDAPSyncerConsumer } = require('../../../../src/services/sync/strategies/LDAPSyncerConsumer');
+
+const { userModel } = require('../../../../src/services/user/model');
+const accountModel = require('../../../../src/services/account/model');
+
+const { classModel } = require('../../../../src/services/user-group/model');
+const { schoolModel } = require('../../../../src/services/school/model');
+
+const appPromise = require('../../../../src/app');
+
+const testObjects = require('../../helpers/testObjects')(appPromise);
+
+const { expect } = chai;
+chai.use(chaiAsPromised);
+
+const ldapSchoolIDn = 'TEST_SCHOOL_LDAP_DN';
+const ldapClassDn = 'TEST_CLASS_LDAP_DN';
+const ldapUserDn = 'TEST_USER_LDAP_DN';
+const accountUserName = 'TEST_ACCOUNT_UNAME';
+
+describe('Ldap Syncer Consumer Integration', () => {
+	let app;
+	let server;
+
+	before(async () => {
+		app = await appPromise;
+		server = await app.listen(0);
+	});
+
+	afterEach(async () => {
+		await accountModel.deleteOne({ username: accountUserName.toLowerCase() }).lean().exec();
```

Comment:
We could move it in a Promise.all, but since it is only test code it's OK for me.